page3.html

```diff
@@ -42,16 +42,16 @@ <h1>Z U L U</h1>
 	<div class="content">
 		<h3>Top Songs Right Now:</h3>
 		<hr>
-		<div class="image"><img src="images/image_placeholder2.jpg" width="99.7%"></div>
+		<div class="image"><img src="images/image_placeholder3.jpg" width="99.7%"></div>

```

Comment:
new image placeholder added to fit in with the stakeholder feedback recieved for the layout